npm-shrinkwrap.json

```diff
@@ -3926,42 +3926,6 @@
         }
       }
     },
-    "handlebars": {
```

Comment:
This is gonna break node 7. You need complete dep tree. Basically you can just add "dependencies" to "oc-template-handlebars" and nest this snippet here